spec/features/create_asset_with_digitalinstantiation_spec.rb

```diff
@@ -137,50 +133,15 @@
       choose('digital_instantiation_visibility_open')
       expect(page).to have_content('Please note, making something visible to the world (i.e. marking this as Public) may be viewed as publishing which could impact your ability to')
 
-      click_on('Save')
-
-      wait_for(10) { DigitalInstantiation.where(title: digital_instantiation_attributes[:title]).first }
-
-      visit '/'
-      find("#search-submit-header").click
-
-      within('#facets') do
-        # Filter resources types
-        click_on('Type')
-        click_on('Asset')
-      end
-
-      # Expect metadata for Asset to be displayed on the search results page.
-      expect(page).to have_content main_title
-
-      # open asset with detail show
-      click_on main_title
```

Comment:
We're doing that in `spec/features/create_asset_spec.rb`. In fact, it looks like that same testing logic has been copied to several integration specs. We probably only need to check it once, yeah? I just happened to remove it from this spec because it's the one I was working in.

Reply:
sounds good!.